### PR TITLE
KD-0 Database backup script - Refactor to only make a backup to a loc…

### PR DESCRIPTION
**Pull request by kivilahtio**
…al filesystem from various configuration sources.

This script should be triggered remotely via a backup-mechanism and the deliverables
be transported using the same mechanism to a safe backup location.

### CODE REVIEW

**Comment by kivilahtio** [comment]
This behaves the same as the debian/scripts/koha-dump -script used bu the Debian package installer, except it can be integrated to from $ENV and cli parameters and doesn't have the hard-coded path variables than the Debian packaged version.

**Comment by taskula** [comment]
This looks very useful +++

**Comment by kivilahtio** [comment]
Also integrates well with other infrastructure orchestration tools.

File: misc/cronjobs/backup.sh [hunk 1]
```diff
@@ -3,206 +3,254 @@
...
+  if [ ! -z "$ANONYMIZE" ]; then
+    $MYSQLDUMP_CMD > $BACKUP_DIR/$DB_BACKUP_FILE
     rc=$?; if [[ $rc != 0 ]]; then echo "mysqldump failed" 1>&2; exit $rc; fi
 
-    sqlanonymize --in $BACKUPDIR/$KOHA_BACKUP_FILE --out - | gzip -9 > $BACKUPDIR/$KOHA_BACKUP_ANONARCHIVE
+    sqlanonymize --in $BACKUP_DIR/$DB_BACKUP_FILE --out - | gzip -9 > $BACKUP_DIR/$DB_BACKUP_ANONARCHIVE
     #Care about exceptions
     ps=(${PIPESTATUS[@]}) #Why is PIPESTATUS emptied when it gets read once?  Copy it as array
-    if [[ ${ps[0]} != 0 ]]; then
+    if   [[ ${ps[0]} != 0 ]]; then
```
Comment (kohaputti) [inline misc/cronjobs/backup.sh]: This whitespace looks accidental?

**Comment by kohaputti** [comment]
I don't use this feature but I checked the changes and it doesn't do anything malicous and it looks to be improving the backup functionality so looks good to me.

**Pull request merged**